app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -219,6 +220,14 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
 
   const [paymentError, setPaymentError] = useState<string | undefined>(undefined)
   const { LL } = useI18nContext()
+  const { error } = useOnChainTxFeeQuery({
```

Comment:
this is a call to the server. we don;t want that just to get the error

Reply:
I was debugging some variables and the `sendPayment `  function returns the correct gql error, like : 
1. Unable to calculate fee because Amount is too low
2. User tried to pay themselves
These errors are shown when the user clicks on confirm payment button.
So, this may come across as redundant to show the same message again, in the Fee container, and when the user clicks on confirm payment...
Should we disable the confirm payment button on such errors and then just show the error once?

Reply:
|||
| :-------------: | :-------------: |
| <video src="https://user-images.githubusercontent.com/74586135/226171020-b5ab66c0-aadf-4534-9307-e5b99e4256e7.mp4" > | <video src="https://user-images.githubusercontent.com/74586135/226171023-d95e28c2-89bb-4b3f-a91e-a768ba4f1f8d.mp4">

Reply:
This is how it works now